pyTSEB/TSEBConfigFileInterface.py

```diff
@@ -175,10 +175,12 @@ def get_data(self, config_data, is_image):
                                                   float(config_data['G_phase']),
                                                   float(config_data['G_shape'])],
                                                  12.0]
-                elif var_name == "flux_LR_method" and self.params["model"] == "disTSEB":
-                    self.params[var_name] = str(config_data[var_name]).strip('"')
-                elif var_name == "correct_LST" and self.params["model"] == "disTSEB":
-                    self.params[var_name] = int(config_data[var_name])
+                elif var_name == "flux_LR_method":
+                    if self.params["model"] == "disTSEB":
+                        self.params[var_name] = str(config_data[var_name]).strip('"')
+                elif var_name == "correct_LST":
+                    if self.params["model"] == "disTSEB":
+                        self.params[var_name] = int(config_data[var_name])
```

Comment:
Maybe it's nicer like in https://github.com/hectornieto/pyTSEB/commit/ed38e4519c76f920a5cb7f052eee94cfad9e166a

Reply:
I might also give the config parsing an overhaul, so I'll keep it as is for now.